src/features/game/types/craftables.ts

```diff
@@ -10,6 +10,7 @@ export type CraftAction = {
 export type CraftableName = NFT | Tool | SeedName | Food;
 
 export type Craftable = {
+  itemType: "food" | "tool" | "nft" | "crop" | "seed"
```

Comment:
Can `seed` or `crop` be a craftable item. Consider removing from this union.

`itemType: "food" | "tool" | "nft"`